### PR TITLE
Added full-screen button

### PR DESCRIPTION
**Pull request by astu9880**
#11

### CODE REVIEW

**Comment by astu9880** [comment]
**Screenshot Normal View**

**Screenshot Full Screen View**

**Comment by astu9880** [comment]
@omerjerk Please review

File: scripts/main.js [hunk 1]
```diff
@@ -92,4 +94,15 @@ $(document).ready(function () {
         $(this).closest('li').addClass('active');
         init();
     });
+
+    $('#expand').click(function () {
+
+            $('.hideonfull').toggleClass('full');
```
Comment (omerjerk) [inline scripts/main.js]: Please fix the indentation here. Make it 4 characters instead of 8.

Comment (astu9880) [inline scripts/main.js]: Fixed

File: scripts/main.js [hunk 2]
```diff
@@ -25,6 +25,8 @@ function init() {
     });
 
 $(document).ready(function () {
+    var fs=false;                          // boolean for full screen state of id
+
```
Comment (omerjerk) [inline scripts/main.js]: Remove this extra line. Rest all looks perfect. Thanks for the pull request. :)

**Comment by astu9880** [comment]
@omerjerk fixed

File: scripts/main.js (hunk 2 above)
Comment (astu9880) [inline scripts/main.js]: :+1: Also, i would like to propose the feature for downloading code written in the ide as a file.

**Comment by astu9880** [comment]
@omerjerk request to merge and close issue

**Comment by astu9880** [comment]
@prateek27 Please review and merge this

**Comment by astu9880** [comment]
@prateek27  @championswimmer  @omerjerk  added language detection from upload along with full screen support. Resolves issues #11  and #30. Please review. Attatching Screenshots.
a1
a2
a3
**WHEN A CPP FILE IS UPLOADED**
a4

**Comment by astu9880** [comment]
@championswimmer @omerjerk @prateek27 
**This PR is getting way too large. With every new merge to master, conflicts are increasing. Please review and merge this PR.**
Summary of this pull request:
1)  Adds full screen toggle functionality up-to commit 	69e0568 
2)  Adds auto detection of language from the file extension and changes the ace editor's mode accordingly f06980f

**Comment by astu9880** [comment]
@prateek27 bhaiya my PR for full screen was before @sirjan13. Why was this not merged? Looks pretty much the same.

**Comment by championswimmer** [comment]
@astu9880 your PR has conflicts. We cannot merge unless your PR is conflict-free

**Comment by astu9880** [comment]
@championswimmer these conflicts arose after #40 was merged. Which provided the same full screen feature and edited the same divs. My PR was proposed way before #40 but was not acknowledged!

**Pull request closed without merging**